contrib/babelfishpg_tsql/src/err_handler.c

```diff
@@ -131,6 +131,7 @@ is_ignorable_error(int pg_error_code, uint8_t override_flag)
 		case SQL_ERROR_8145:
 		case SQL_ERROR_8146:
 		case SQL_ERROR_213:
+		case SQL_ERROR_6615:			
```

Comment:
If we are mapping a new error, we would like to add the test files generated by BQ during their time. Please sync with Dipesh on this